packages/gatsby-source-wordpress/src/steps/create-schema-customization/index.js

```diff
@@ -67,6 +67,11 @@ const customizeSchema = async ({ actions, schema, store: gatsbyStore }) => {
           break
       }
 
+      if (type.interfaces && builtType?.config) {
+        builtType.config.interfaces ||= []
+        builtType.config.interfaces.push(...buildInterfacesListForType(type))
+      }
```

Comment:
All interfaces on a type from the origin schema need to be brought into types in the Gatsby schema for it to be valid now